postman/scripts/runPostman.ts

```diff
@@ -145,8 +145,11 @@ async function main() {
         ? parseInt(process.env.DB_DAYS_BEFORE_NOW_TO_DELETE)
         : undefined,
     },
+    apiOptions: {
+      port: process.env.API_PORT ? parseInt(process.env.API_PORT) : undefined,
```

Comment:
Should we add `API_PORT` variable to `docker/config/postman/env` for the published Postman image?